blockchain/blockchain.go

```diff
@@ -174,6 +174,9 @@ type BlockChain struct {
 	// Warm up
 	lastCommittedBlock uint64
 	quitWarmUp         chan struct{}
+
+	// InternalTx Tracing Buffer
+	internalTraces []*vm.InternalTxTrace
```

Comment:
This `internalTraces` can be conflict with traceAPIs. 
So Why don't you make `ApplyTransaction` returns a log and `Process` gathers and returns the logs.

Reply:
Ok. I will change it. But, what do you mean conflicting with traceAPIs?

Reply:
0c4e033fdf5b49f0743accdbaa3e8c3bf1d07baf